{"team1": {"ubuntu1ping": {"error": "-c: 1: ping: not found\n", "score": 0}, "ubuntu2ping": {"error": "-c: 1: ping: not found\n", "score": 0}, "ubuntu1ssh": {"error": "Not tested", "score": 0}, "ubuntu2ssh": {"error": "Not tested", "score": 0}, "ubuntu1web": {"error": "Not tested", "score": 0}, "ubuntu2web": {"error": "Not tested", "score": 0}}, "team2": {"ubuntu1ping": {"error": "-c: 1: ping: not found\n", "score": 0}, "ubuntu2ping": {"error": "-c: 1: ping: not found\n", "score": 0}, "ubuntu1ssh": {"error": "[Errno None] Unable to connect to port 22 on 10.0.2.20", "score": 0}, "ubuntu2ssh": {"error": "[Errno None] Unable to connect to port 22 on 10.0.2.30", "score": 0}, "ubuntu1web": {"error": "HTTPConnectionPool(host='10.0.2.20', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7102a41990a0>: Failed to establish a new connection: [Errno 113] EHOSTUNREACH'))", "score": 0}, "ubuntu2web": {"error": "HTTPConnectionPool(host='10.0.2.30', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7102a4198ce0>: Failed to establish a new connection: [Errno 113] EHOSTUNREACH'))", "score": 0}}}